src/Cli/dotnet/commands/dotnet-workload/config/WorkloadConfigCommand.cs

```diff
@@ -68,7 +68,11 @@ public override int Execute()
                 }
                 else if (string.IsNullOrEmpty(_updateMode))
                 {
-                    if (InstallingWorkloadCommand.ShouldUseWorkloadSetMode(_sdkFeatureBand, _dotnetPath))
+                    string globalJsonPath = SdkDirectoryWorkloadManifestProvider.GetGlobalJsonPath(Environment.CurrentDirectory);
```

Comment:
Note to self: I should probably turn this into errors if it doesn't align properly